templates/react/components/foo/List.js

```diff
@@ -84,4 +77,13 @@ const mapDispatchToProps = (dispatch) => {
   };
 };
 
+List.propTypes = {
```

Comment:
React.PropTypes is deprecated as of React v15.5
https://facebook.github.io/react/docs/typechecking-with-proptypes.html